### ISSUE TITLE
NullPointerException when calling getMaterial on ChunkSnapshot

### ISSUE THREAD

**Issue by ChristopherHaws** [MockBukkit/MockBukkit#660]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest version of MockBukkit?

- [X] I am using the latest version of MockBukkit.

### Minecraft Version

1.19

### Describe the bug

I have a plugin that creates a chunk snapshot, passes it to a new thread, and calls getBlockType on the chunk snapshot and I am getting the following error from MockBukkit:
```
Exception in thread "Thread-4" java.lang.NullPointerException: Cannot invoke "org.bukkit.block.data.BlockData.getMaterial()" because the return value of "java.util.Map.get(Object)" is null
    at be.seeseemelk.mockbukkit.ChunkSnapshotMock.getBlockType(ChunkSnapshotMock.java:61)
```

### Reproducible Test

This is the test I am trying to write that is having the error:
https://github.com/ChristopherHaws/mc-automatic-inventory/blob/11a04f647f4076f868761c2c6cbdc1474e039f5b/src/test/java/DepositAllEndToEndTests.java#L30

**Comment by thelooter** [comment]
Can you give us the whole Stacktrace?

**Comment by Insprill** [comment]
And what method are you using to get the snapshot?

**Comment by Insprill** [comment]
What method are you using to create the snapshot?

**Comment by Insprill** [comment]
What method are you using to create the snapshot?

**Comment by Insprill** [comment]
~~Closed in #661~~

**Comment by ChristopherHaws** [comment]
Hey there! Sorry for not putting the whole stacktrace. For the record, this was the full stacktrace:
```
Exception in thread "Thread-4" java.lang.NullPointerException: Cannot invoke "org.bukkit.block.data.BlockData.getMaterial()" because the return value of "java.util.Map.get(Object)" is null
	at be.seeseemelk.mockbukkit.ChunkSnapshotMock.getBlockType(ChunkSnapshotMock.java:61)
	at dev.chaws.automaticinventory.tasks.AsyncChestDepositTask.getType(AsyncChestDepositTask.java:108)
	at dev.chaws.automaticinventory.tasks.AsyncChestDepositTask.run(AsyncChestDepositTask.java:59)
```

I will give the latest version a try and let you know how that goes! Thanks for the the quick turnaround! :)
